src/utils/utils.ts

```diff
@@ -58,6 +58,19 @@ export const loadStyles = async (url: string): Promise<void> => {
   })
 }
 
+export const loadHandlebars = (): void => {
+  const handlebarScript = document.createElement('script')
+  handlebarScript.src =
+    'https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.7/handlebars.min.js'
+  document.head.appendChild(handlebarScript)
+
+  handlebarScript.addEventListener('load', () => {
+    const assignScript = document.createElement('script')
+    assignScript.textContent = ' window.handlebarsForHummWidget = Handlebars '
+    document.head.appendChild(assignScript)
+  })
+}
```

Comment:
Really important!
This is how I'm giving the widget access to `Handlebars` via a CDN.
Within the widget, handlebar can be accessed via `window. handlebarsForHummWidget`.